packages/vkui/src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -83,7 +92,7 @@ export const SegmentedControl = ({
         size === 'l' && styles.sizeL,
       )}
     >
-      <div role="radiogroup" className={styles.in}>
+      <div role={role === 'tablist' ? 'tablist' : 'radiogroup'} ref={tabsRef} className={styles.in}>
```

Comment:
```suggestion
      <div role={role} ref={tabsRef} className={styles.in}>
```
вижу `role` по умолчанию `'radiogroup'`